core-bundle/src/Resources/config/services.yml

```diff
@@ -407,6 +407,7 @@ services:
 
     contao.mailer.available_transports:
         class: Contao\CoreBundle\Mailer\AvailableTransports
+        public: true
```

Comment:
_Note:_ this isn't strictly necessary as the currently used annotation will make the service public in any case. However, since we use the service directly in a DCA in a completely different bundle, I think it makes sense to specifically define the service as being public, instead of relying on the annotation (as the annotation only works within the `core-bundle`).